src/main/java/net/testudobank/MvcController.java

```diff
@@ -183,6 +183,9 @@ private static Date convertLocalDateTimeToDate(LocalDateTime ldt){
     return dateTime;
   }
 
+  private double applyInterestRateToPennyAmount(int pennyAmount){
+    return INTEREST_RATE*pennyAmount;
```

Comment:
Make this return an `int` and make the method logic be `(int) (INTEREST_RATE * pennyAmount)` instead so that the caller of this function doesn't have to worry about remembering to add the `(int)` cast themselves.